<?xml version="1.0" encoding="UTF-8" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/">
  <channel>
    <title>Server rune error [delta] 317</title>
    <link>https://forum.moparisthebest.com/t/server-rune-error-delta-317/544082</link>
    <description>hey.. I got this when I ran my sever.

[i]Loading npcs... drops... spawn list...done!
com.mysql.jdbc.CommunicationsException: Communications link failure due to underlying exception: 

** BEGIN NESTED EXCEPTION ** 

java.net.ConnectException
MESSAGE: Connection refused: connect

STACKTRACE:

java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.&lt;init&gt;(Unknown Source)
	at java.net.Socket.&lt;init&gt;(Unknown Source)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:173)
	at com.mysql.jdbc.MysqlIO.&lt;init&gt;(MysqlIO.java:267)
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2739)
	at com.mysql.jdbc.Connection.&lt;init&gt;(Connection.java:1553)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:266)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at SQL.createConnection(SQL.java:14)
	at server.run(server.java:255)
	at java.lang.Thread.run(Unknown Source)


** END NESTED EXCEPTION **



Last packet sent to the server was 31 ms ago.
	at com.mysql.jdbc.Connection.createNewIO(Connection.java:2814)
	at com.mysql.jdbc.Connection.&lt;init&gt;(Connection.java:1553)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:266)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at SQL.createConnection(SQL.java:14)
	at server.run(server.java:255)
	at java.lang.Thread.run(Unknown Source)
Server is already in use.
[/i]</description>
    
    <lastBuildDate>Fri, 20 Jun 2014 23:31:20 +0000</lastBuildDate>
    <category>Runescape</category>
    <atom:link href="https://forum.moparisthebest.com/t/server-rune-error-delta-317/544082.rss" rel="self" type="application/rss+xml" />
      <item>
        <title>Server rune error [delta] 317</title>
        <dc:creator><![CDATA[@The_Web_Guy The Web Guy]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/the_web_guy">@The_Web_Guy</a> wrote:</p>
          <blockquote>
              <aside class="quote" data-post="2" data-topic="544082">
<div class="title">
<div class="quote-controls"></div>
<img alt width="20" height="20" src="https://forum.moparisthebest.com/letter_avatar/aeonscape/40/5_e05bb34c421432ee4d40de30c10af3e5.png" class="avatar"> Aeonscape:</div>
<blockquote>
<p>hi , have you edited any npc drop or npc spawns the code might be out line with what it should be.</p>
</blockquote>
</aside>
<p>…That has nothing to do with it.</p>
<p>This means that the server is trying to connect to a SQL database, but cannot. Go into server.java and check to see if it’s trying to connect to SQL. If you don’t know what to look for, paste your server.java here.</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/server-rune-error-delta-317/544082/3">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/server-rune-error-delta-317/544082/3</link>
        <pubDate>Fri, 20 Jun 2014 23:31:20 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-544082-3</guid>
        <source url="https://forum.moparisthebest.com/t/server-rune-error-delta-317/544082.rss">Server rune error [delta] 317</source>
      </item>
      <item>
        <title>Server rune error [delta] 317</title>
        <dc:creator><![CDATA[@Aeonscape Aeonscape]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/aeonscape">@Aeonscape</a> wrote:</p>
          <blockquote>
              <p>hi , have you edited any npc drop or npc spawns the code might be out line with what it should be.</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/server-rune-error-delta-317/544082/2">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/server-rune-error-delta-317/544082/2</link>
        <pubDate>Fri, 20 Jun 2014 16:45:28 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-544082-2</guid>
        <source url="https://forum.moparisthebest.com/t/server-rune-error-delta-317/544082.rss">Server rune error [delta] 317</source>
      </item>
      <item>
        <title>Server rune error [delta] 317</title>
        <dc:creator><![CDATA[@DaGleesh DaGleesh]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/dagleesh">@DaGleesh</a> wrote:</p>
          <blockquote>
              <p>hey… I got this when I ran my sever.</p>
<p>[i]Loading npcs… drops… spawn list…done!<br>
com.mysql.jdbc.CommunicationsException: Communications link failure due to underlying exception:</p>
<p>** BEGIN NESTED EXCEPTION **</p>
<p>java.net.ConnectException<br>
MESSAGE: Connection refused: connect</p>
<p>STACKTRACE:</p>
<p>java.net.ConnectException: Connection refused: connect<br>
at java.net.DualStackPlainSocketImpl.connect0(Native Method)<br>
at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)<br>
at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)<br>
at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)<br>
at java.net.AbstractPlainSocketImpl.connect(Unknown Source)<br>
at java.net.PlainSocketImpl.connect(Unknown Source)<br>
at java.net.SocksSocketImpl.connect(Unknown Source)<br>
at java.net.Socket.connect(Unknown Source)<br>
at java.net.Socket.connect(Unknown Source)<br>
at java.net.Socket.(Unknown Source)<br>
at java.net.Socket.(Unknown Source)<br>
at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:173)<br>
at com.mysql.jdbc.MysqlIO.(MysqlIO.java:267)<br>
at com.mysql.jdbc.Connection.createNewIO(Connection.java:2739)<br>
at com.mysql.jdbc.Connection.(Connection.java:1553)<br>
at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:266)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at SQL.createConnection(SQL.java:14)<br>
at server.run(server.java:255)<br>
at java.lang.Thread.run(Unknown Source)</p>
<p>** END NESTED EXCEPTION **</p>
<p>Last packet sent to the server was 31 ms ago.<br>
at com.mysql.jdbc.Connection.createNewIO(Connection.java:2814)<br>
at com.mysql.jdbc.Connection.(Connection.java:1553)<br>
at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:266)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at java.sql.DriverManager.getConnection(Unknown Source)<br>
at SQL.createConnection(SQL.java:14)<br>
at server.run(server.java:255)<br>
at java.lang.Thread.run(Unknown Source)<br>
Server is already in use.<br>
[/i]</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/server-rune-error-delta-317/544082/1">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/server-rune-error-delta-317/544082/1</link>
        <pubDate>Wed, 18 Jun 2014 16:14:05 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-544082-1</guid>
        <source url="https://forum.moparisthebest.com/t/server-rune-error-delta-317/544082.rss">Server rune error [delta] 317</source>
      </item>
  </channel>
</rss>
